app/services/event_generator/geoblacklight_event_generator.rb

```diff
@@ -36,7 +36,7 @@ def record_member_updated(record)
         record_deleted(record)
       elsif state == "complete"
         publish_message(
-          message("MEMBER_UPDATED", record)
+          message("UPDATED", record)
```

Comment:
Why not just call `record_updated(record)` in this method? Or get geoblacklight to read MEMBER_UPDATED events.